src/iOS/Xamarin.iOS/Samples/Search/FindPlace/FindPlace.cs

```diff
@@ -150,158 +77,148 @@ public override void ViewDidLoad()
-
-            // Set the default location and search text
-            _myLocationBox.Text = "Current Location";
-            _mySearchBox.Text = "Coffee";
-
-            // Allow pressing 'return' to dismiss the keyboard
-            _myLocationBox.ShouldReturn += (textField) => { textField.ResignFirstResponder(); return true; };
-            _mySearchBox.ShouldReturn += (textField) => { textField.ResignFirstResponder(); return true; };
-
-            // Gray out the buttons when they are disabled
-            _mySearchButton.SetTitleColor(UIColor.Gray, UIControlState.Disabled);
-            _mySearchRestrictedButton.SetTitleColor(UIColor.Gray, UIControlState.Disabled);
-
-            // Change button color when enabled
-            _mySearchButton.SetTitleColor(UIColor.Blue, UIControlState.Normal);
-            _mySearchRestrictedButton.SetTitleColor(UIColor.Blue, UIControlState.Normal);
-
-            // Color the textboxes and buttons to appear over the mapview
-            UIColor background = UIColor.FromWhiteAlpha(.85f, .95f);
-            _mySearchBox.BackgroundColor = background;
-            _myLocationBox.BackgroundColor = background;
-            _mySearchButton.BackgroundColor = background;
-            _mySearchRestrictedButton.BackgroundColor = background;
-            _myProgressBar.BackgroundColor = background;
-
-            // Hide the activity indicator (progress bar) when stopped
-            _myProgressBar.HidesWhenStopped = true;
-
-            // Set radii to make it look nice
-            _mySearchRestrictedButton.Layer.CornerRadius = 5;
-            _mySearchButton.Layer.CornerRadius = 5;
-            _myLocationBox.Layer.CornerRadius = 5;
-            _mySearchBox.Layer.CornerRadius = 5;
-
-            // Make sure the suggestion list is hidden by default
-            _mySuggestionView.Hidden = true;
-
-            // Create the suggestion source
+            // Create the suggestion source.
             _mySuggestionSource = new SuggestionSource(null, this);
 
-            // Set the source for the table view
-            _mySuggestionView.Source = _mySuggestionSource;
+            // Set the source for the table view.
+            _suggestionView.Source = _mySuggestionSource;
 
-            // Enable tap-for-info pattern on results
-            _myMapView.GeoViewTapped += _myMapView_GeoViewTapped;
+            // Enable tap-for-info pattern on results.
+            _myMapView.GeoViewTapped += MapView_GeoViewTapped;
 
-            // Listen for taps on the search buttons
-            _mySearchButton.TouchUpInside += _mySearchButton_Clicked;
-            _mySearchRestrictedButton.TouchUpInside += _mySearchRestrictedButton_Click;
+            // Listen for taps on the search buttons.
+            _searchButton.TouchUpInside += SearchButton_Touched;
+            _searchRestrictedButton.TouchUpInside += SearchRestrictedButton_Touched;
```

Comment:
I'm noticing that once the suggestion list pops up, it can get "stuck" open -- even after I use "return" on the keyboard, after I tap one of the Search buttons, or even after I interact with the MapView. Example:

<img src="https://user-images.githubusercontent.com/587809/41946115-933c4694-7965-11e8-8713-463a29dab33c.jpeg" width="300" />